### ISSUE TITLE
NSFW GIFs don’t load.

### ISSUE THREAD

**Issue by emansanpacion1** [christianselig/apollo-bugs#16]
**Describe the bug**
NSFW GIFs are stuck on an infinite load, even when app is restarted.

**To Reproduce**
1. Go to a sub like r/RedditorCum (Sorry)
2. Click on GIF; try 3D Touch-ing on a GIF and it will not load.

**iOS version**: 12.2
**Apollo version**: 1.4.18
**iOS device**: iPhone 8 Plus
**Jailbroken**: No
**Reproducible:** Every time

**Comment by Riichie96** [comment]
Yes, I also have this. And weirdly enough it's only present on NSFW GIF's. Normal GIF's work fine.

**Comment by that-knave** [comment]
I don’t know that this is specifically a NSFW gif issue so much as a gyfcat issue. All of the gifs I’ve seen in infinite black screen have been uploaded to gyfcat.
